tests/functional/test.cli.run.js

```diff
@@ -1,14 +1,21 @@
 /* @flow */
-import {describe, it} from 'mocha';
+import {describe, it, before} from 'mocha';
 
 import {
-  webExt, addonPath, fakeFirefoxPath,
-  withTempAddonDir, execCommand, reportCommandErrors,
+  addonPath, fakeFirefoxPath,
+  withTempAddonDir, execWebExt, reportCommandErrors,
 } from './common';
 
 const EXPECTED_MESSAGE = 'Fake Firefox binary executed correctly.';
 
 describe('web-ext run', () => {
+  before(function() {
+    // you can't spawn .js file as an executable on Windows
```

Comment:
would it be possible to put a `fake-firefox-binary.bat` file in the same directory that simply executes `fake-firefox-binary.js`? If it's a hassle, we could leave it as a skip and file an issue to fix it later.